setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
+
+setup(
+    name='Embedded2',
+    version='0.5.0',
+    description='Fancy goggle detector',
```

Comment:
I would say 'Face detection and goggle classification'

Reply:
Also a placeholder but I can make that change